GitUI/UserControls/FileStatusList.cs

```diff
@@ -17,7 +17,8 @@
 
 namespace GitUI
 {
-    using GitItemsWithParents = IDictionary<string, IList<GitItemStatus>>;
+    //Parents is used as the "first selected" (not always the parent) for GitItemStatus
+    using GitItemsWithParents = IDictionary<GitRevision, IList<GitItemStatus>>;
```

Comment:
this doesn't seem to be consistently used, perhaps remove it completely

Reply:
This PR enables the possibility to use the parent, not always set before. The use of the parent is in follow up PRs.
I tried to make the PRs possible to review but submitted the complete solution as "wip".

Reply:
My comment was about the type alias `GitItemsWithParents` - in other places in the code `IDictionary<GitRevision, IList<GitItemStatus>>` is still widely used

Reply:
Can change to consistently use GitItemsWithParents, IGitItemsWithParents (will have to add that). Or should the alias be removed?

Reply:
I don't mind either way - as long as it consistent :)